[all hosts] (Office 2013) Updating to reflect Office 2013 end-of-support (#3931)
OfficeDev/office-js-docs-pr: docs/develop/application-specific-api-model.md

This article describes how to use the API model for building add-ins in Excel, Word, PowerPoint, and OneNote. It introduces core concepts that are fundamental to using the promise-based APIs.

> [!NOTE]

> > This model is not supported by Outlook or Project clients. Use the [Common API model](office-javascript-api-object-model.md) to work with those applications. For full platform availability notes, see [Office client application and platform availability for Office Add-ins](/javascript/api/requirement-sets).

Reviewer: ```suggestion
> This model isn't supported by Outlook or Project clients. Use the [Common API model](office-javascript-api-object-model.md) to work with those applications. For full platform availability notes, see [Office client application and platform availability for Office Add-ins](/javascript/api/requirement-sets).
```
